package.json

```
@@ -34,6 +34,7 @@
   },
   "homepage": "https://rakuten-rex.github.io/switch/",
   "dependencies": {
+    "@rakuten-rex/grid": "^1.4.0",
```

once you add or modify dependencies, make sure to remove the automatic `^`

```suggestion
    "@rakuten-rex/grid": "1.4.0",
```